test/__snapshots__/test.js.snap

```diff
@@ -862,6 +1181,103 @@ const Impl = require(\\"../implementations/Global.js\\");
 "
 `;
 
+exports[`HTMLConstructor.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+// HTMLConstructor
```

Comment:
According to how https://heycam.github.io/webidl/#overridden-constructor-steps is used (as part of the definition for `[HTMLConstructor]`), it seems like it would be more appropriate to pass the body of the constructor to the hook rather than the entire interface. However happy to have this if it turns out to be necessary for the implementation of CE in jsdom.

Reply:
Agreed, passing the constructor would be ideal. 

In the case `[HTMLConstructor]` a new constructor is needed per realm to capture the global object. In order to get around this, I inject an install method to the exported `iface` object. This is why the entire module is passed and not only the constructor.

You can see how this hook is used on the CE PR: https://github.com/jsdom/jsdom/pull/2548/files#diff-4c420ef958793615b9bf0a6fd47a1fe1R29-R50

Reply:
Great. Thanks for explaining.